projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -977,4 +987,5 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   disableClientTokens: false,
   incrementProcessesCountForMergeCart: true,
   dispatchLoginActionOnlyWhenTokenReceived: false,
+  nativeSupportForRedirectingOAuthFlows: true,
```

Comment:
`major:` To not provide breaking changes for current customers, let's change its value to `false`
When it comes to fresh apps, there is a [special script](https://github.com/SAP/spartacus/blob/f28330fc71c03d2e7841823e445a99e71b9ebf7d/projects/schematics/src/add-spartacus/spartacus-feature-toggles.ts#L26) that sets all flags to `true` when scaffolding the app
```suggestion
  nativeSupportForRedirectingOAuthFlows: false,
```